components/ContributionForm.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
I don't think this component is still used. If not, please remove.

Reply:
Yes right i will delete it